### PR TITLE
Improved documentation

### PR DESCRIPTION
**Pull request by fabiodcasilva**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,31 +2,26 @@
 
 ### 1.5 (Dec 11, 2015)
 
--   Pipeline support
--   Requires 1.609 or higher of Jenkins.
+- Support for [**Pipeline**](https://github.com/jenkinsci/pipeline-plugin).
```
Comment (martinda) [inline CHANGELOG.md:5]: I would not remove the information about the minimum core version requirement 1.609 for this version of the plugin.

**Review by martinda — CHANGES_REQUESTED** [review]
I like the documentation improvements. Thank you! If you can please address the one detail that would be swell.

File: CHANGELOG.md (hunk 1 above)
Comment (fabiodcasilva) [reply CHANGELOG.md:5]: Thanks! It makes sense to let users known about it.

**Review by martinda — APPROVED** [review]
Thank you @fabiodcasilva .

**Pull request merged**